Commands/Commands/queue.js

```diff
@@ -370,6 +370,9 @@ commands.registerVote = {
                     cause: 'chat_vote',
                     message: (e.message !== undefined) ? e.message : JSON.stringify(e)
                   }, e)
+                } else if (e.statusCode === 422) {
```

Comment:
And I was saying you could have done that on my pr, anyhow, we do this sort of thing on a lot of our commands, I wonder if there is something we can put in rather then these if statements everywhere? Of course This for example would need its own thing as its a "special" case.